CHANGELOG.md

```diff
@@ -1,8 +1,32 @@
 ## Change Log
 
-### v3.1.0-preview10 - 2020-06-09
+### v3.1.0-preview10 - 2020-08-12
 
+* Improve error handling during message parsing.
+* Add `MessageParseException` class.
 * Added `CauseOfDeathEntityAxisList` property to `CodingResponseMessage` and `CodingUpdateMessage`.
+* Updated `DeathRecord` class to match latest VRDR IG changes.
+* Numerous under-the-covers changes to address IG changes from FHIR STU3 -> R4 and fix errors and warnings from the FHIR validator.
+* Added `DeathRecord.StateLocalIdentifier` property.
+* `DeathRecord.InterestedPartyIdentifier` is now a `Dictionary` instead of a `string`.
+* Added `DeathRecord.CertifierIdentifier` property.
+* `DeathRecord.ResidenceWithinCityLimits` is now a `Dictionary` instead of a `bool`.
+* Added `DeathRecord.ResidenceWithinCityLimitsBoolean` property as a convenience method.
+* Added `DeathRecord.UsualOccupationEnd` property.
+* Added `DeathRecord.MilitaryServiceBoolean` property as a convenience method.
+* `DeathRecord.MorticianIdentifier` is now a `Dictionary` instead of a `string`.
+* Added `DeathRecord.FuneralDirectorPhone` property.
+* Added `DeathRecord.AutopsyPerformedBoolean` property as a convenience method.
+* Added death pronouncer: `PronouncerGivenNames`, `PronouncerFamilyName`, `PronouncerSuffix`, `PronouncerIdentifier`.
+* Added `DeathRecord.DateOfDeath` property.
+* Added `DeathRecord.AutopsyResultsAvailableBoolean` property as a convenience method.
+* Added `DeathRecord.DeathLocationType` property.
+* `DeathRecord.ExaminerContacted` is now a `Dictionary` instead of a `bool`.
+* Added `DeathRecord.ExaminerContactedBoolean` property as a convenience method.
+* `DeathRecord.InjuryPlace` is now a `Dictionary` instead of a `string`.
```

Comment:
This might be an issue, in that the IJE specifies the place of injury as a literal string (POILITRL) and there's likely a lot of data that probably won't be able to fit into a pretty narrow value set... an issue to bring up with AMS I think

Reply:
There's a separate InjuryLocationDescription property that is a string description. InjuryPlace is the type of location as a code.